src/osdag/setup.py

```diff
@@ -0,0 +1,17 @@
+# setup.py
```

Comment:
We're already using `pyproject.toml`, which is a more modern method. Probably safe to delete.